mllib/src/main/scala/org/apache/spark/ml/image/ImageSchema.scala

```diff
@@ -37,20 +37,67 @@ import org.apache.spark.sql.types._
 @Since("2.3.0")
 object ImageSchema {
 
-  val undefinedImageType = "Undefined"
+  /**
+   * OpenCv type representation
+   *
+   * @param mode ordinal for the type
+   * @param dataType open cv data type
```

Comment:
small nitpick: I think we should always spell it as "OpenCV" to be consistent in the comments (unless you have any good objections)